### PR TITLE
chore: bump to Bazel 7

### PR DESCRIPTION
**Pull request by gregmagolan**
Pre-factor from https://github.com/aspect-build/rules_webpack/pull/146.

### CODE REVIEW

File: webpack/private/is_bazel_7_or_greater.bzl [hunk 1]
```diff
@@ -0,0 +1,7 @@
+"is_bazel_7_or_greater"
+
+def is_bazel_7_or_greater():
```
Comment (gregmagolan) [inline webpack/private/is_bazel_7_or_greater.bzl:3]: This file will be removed once on rules_js 2.0 and with a minimum of bazel-lib 2.x

File: internal_deps.bzl [hunk 2]
```diff
@@ -20,20 +20,20 @@ def rules_webpack_internal_deps():
 
     http_archive(
         name = "io_bazel_rules_go",
-        sha256 = "099a9fb96a376ccbbb7d291ed4ecbdfd42f6bc822ab77ae6f1b5cb9e914e94fa",
-        urls = ["https://github.com/bazelbuild/rules_go/releases/download/v0.35.0/rules_go-v0.35.0.zip"],
+        sha256 = "80a98277ad1311dacd837f9b16db62887702e9f1d1c4c9f796d0121a46c8e184",
```
Comment (alexeagle) [inline internal_deps.bzl:23]: blah! we should remove go from this repo, we have no Go sources, and `aspect configure` is a statically-linked binary

File: webpack/private/is_bazel_7_or_greater.bzl (hunk 1 above)
Comment (alexeagle) [reply webpack/private/is_bazel_7_or_greater.bzl:3]: Is the principled answer to use `bazel_features` and be conditional for this?

**Review by alexeagle — APPROVED** [review]

File: webpack/private/is_bazel_7_or_greater.bzl (hunk 1 above)
Comment (gregmagolan) [reply webpack/private/is_bazel_7_or_greater.bzl:3]: Using it for presence of cache.bzl . Nothing in bazel_features for that.

File: internal_deps.bzl (hunk 2 above)
Comment (gregmagolan) [reply internal_deps.bzl:23]: follow-up

**Pull request merged**